mozilla/blurts-server
# MNTOR-1271: Extend errors classes

# References: 
Jira: [MNTOR-1271](https://mozilla-hub.atlassian.net/browse/MNTOR-1271)



# Description

Update `Errors` module to be central store for serving unified error messages.

# How to test

Since this PR is replacing error classes across the codebase we just should make sure, that the error types that are being replaced are the ones we would expect to be thrown.

For testing the client-side error that is showing a toast notification optionally one could invert the condition [in this line](https://github.com/mozilla/blurts-server/pull/3010/files#diff-135402cfb34bfc8e8f3df69241e60d14eec552b4edfdbbf87996209784f60c4aR59) and trigger a modal.

# Checklist (Definition of Done)
